### DISCUSSION TITLE
[Art competition] Correctness

### DISCUSSION THREAD

**Discussion by Gimongjin** [prosyslab-classroom/cs424-program-reasoning#259] [Art competition]
![DALL·E 2022-11-30 22 23 10 - a precious blue diamond hidden deep and hard to find](https://user-images.githubusercontent.com/79095967/204810371-9cc90487-e335-470e-add9-bde83b99097e.png)

## Short Description
For me, Correctness means a precious blue diamond hidden deep and hard to find".

## Long Description (up to 3 sentences)
The correctness of a program is very important concept for programmers – program with proven correctness has a very strong power. But correctness of program is very hard to achieve, so we try to use so many kinds of methods to find it from various program spaces. Actually, we don’t even know whether it is existing or not! - But we are still taking many adventures to find the correctness.

**Comment by KihongHeo** [comment]
Beautiful.
